crates/e2e/src/setup/services.rs

```diff
@@ -23,6 +23,7 @@ use {
 
 pub const API_HOST: &str = "http://127.0.0.1:8080";
 pub const ORDERS_ENDPOINT: &str = "/api/v1/orders";
+pub const ORDER_STATUS_ENDPOINT: &str = "/api/v1/status";
```

Comment:
I just thought about this.

Would it make sense to put this as part of the orders endpoint?
After all, we are querying an order status.
I mean this: `/api/v1/orders/<id>/status`

Unless we expect to extend this in the future for trades and so on, then it's all good.

Feel free to disregard.

Reply:
Makes sense, thanks. Migrated.